Makefile

```diff
@@ -27,26 +29,24 @@ stop: ## stop docker containers
 dc_build: ## rebuild docker compose containers
 	- docker-compose up --build -d
 
+purge: ## cleaning
+	- rm -rf node_modules vendor var/cache var/log public/build
+
+status: ## docker containers status
+	- docker-compose ps
+
+##@ DEV
+
 install: ## install php and node dependencies
-	- docker-compose exec phpfpm composer install
+	- docker-compose exec phpfpm composer install --ignore-platform-reqs
```

Comment:
🤔 ... code smell? could you articulate the reasons? In my experience this kind of "assumptions" could hide bugs..

Reply:
My bad, that slipped in, it wasn't supposed to be committed. Although, it is useful for php8 since most of the dependencies are not supporting it on their composer side.

Reply:
then leave it alone in the job dedicated to php 8

Reply:
Done